src/wiki/development/development-builds.md

```diff
@@ -17,6 +17,4 @@ Portable dev-builds are also provided; for both Windows and Linux.
 
 For Arch, Debian and Gentoo, respectively, you can use these packages for the latest development versions:
 
-[![polymc-git](https://img.shields.io/badge/aur-polymc--git-blue)](https://aur.archlinux.org/packages/polymc-git/)   [![polymc-git](https://img.shields.io/badge/mpr-polymc--git-orange)](https://mpr.makedeb.org/packages/polymc-git)   [![polymc-9999](https://img.shields.io/badge/gentoo-polymc--9999-purple)](https://packages.gentoo.org/packages/games-action/polymc)
-
-For Flatpak builds, you can use [flathub-beta](https://discourse.flathub.org/t/how-to-use-flathub-beta/2111).
+[![polymc-git](https://img.shields.io/badge/aur-polymc--git-blue)](https://aur.archlinux.org/packages/polymc-qt5-git/) [![polymc-git](https://img.shields.io/badge/aur-polymc--qt5--git-blue)](https://aur.archlinux.org/packages/polymc-git/)    [![polymc-git](https://img.shields.io/badge/mpr-polymc--git-orange)](https://mpr.makedeb.org/packages/polymc-git) [![polymc-9999](https://img.shields.io/badge/gentoo-polymc--9999-purple)](https://packages.gentoo.org/packages/games-action/polymc)
```

Comment:
wait there's another one between the debian things, i havent seen this one lol

Reply:
fixed